index.html

```diff
@@ -1727,12 +1727,13 @@ <h3>Issuer</h3>
         <dl>
           <dt><var id="defn-issuer">issuer</var></dt>
           <dd>
-The value of the <code>issuer</code> <a>property</a> MUST be either a
-<a>URL</a> or an object containing an <code>id</code> <a>property</a>. It is
-RECOMMENDED that the <a>URL</a> in the <code>issuer</code> or its
-<code>id</code> be one which, if dereferenced, results in a document containing
-machine-readable information about the <a>issuer</a> that can be used to
-<a>verify</a> the information expressed in the <a>credential</a>.
+The value of the <code>issuer</code> <a>property</a> MUST be a
+<a>URL</a>, or an object containing an <code>id</code> <a>property</a> whose
+value is a <a>URL</a>, which identifies the issuer in a globally unambiguous
```

Comment:
and we are asserting that DIDs are URLs here right?

Reply:
Well, the DID spec asserts that a DID URL is a URL. This text just says "any URL will do"... so, yes, a DID is a valid value here (as is any HTTP URL).